examples/react/kitchen-sink/src/main.tsx

```diff
@@ -180,7 +181,7 @@ const loginRoute = rootRoute.createRoute({
 })
 
 const layoutRoute = rootRoute.createRoute({
-  id: 'layout',
+  path: 'layout',
```

Comment:
doesn't this defeat the purpose of the layout route in the example?
I thought a layoutRoute didn't have a path by design.

https://tanstack.com/router/v1/docs/guide/layout-routes